packages/webdriver/src/monad.js

```diff
@@ -68,30 +77,69 @@ export default function WebDriver (options, modifier, propertiesObject = {}) {
                 ? commandWrapper(name, func)
                 : func
             if (attachToElement) {
+                /**
+                 * add command to every multiremote instance
+                 */
                 if (instances) {
                     Object.values(instances).forEach(instance => {
                         instance.__propertiesObject__[name] = {
                             value: customCommand
                         }
                     })
                 }
+
                 this.__propertiesObject__[name] = { value: customCommand }
             } else {
                 unit.lift(name, customCommand, proto)
             }
         }
 
+        /**
+         * overwriteCommand
+         * @param  {String}   name              command name to be overwritten
+         * @param  {Function} func              function to replace original command with;
+         *                                      takes original function as first argument.
+         * @param  {boolean=} attachToElement   overwrite browser command (false) or element command (true)
+         * @param  {Object=}  proto             prototype to add function to (optional)
+         * @param  {Object=}  instances         multiremote instances
+         */
+        client.overwriteCommand = function (name, func, attachToElement = false, proto, instances) {
+            let customCommand = typeof commandWrapper === 'function'
+                ? commandWrapper(name, func)
+                : func
+            if (attachToElement) {
+                if (instances) {
+                    /**
+                     * add command to every multiremote instance
+                     */
+                    Object.values(instances).forEach(instance => {
+                        instance.__propertiesObject__.__elementOverrides__.value[name] = customCommand
+                    })
+                } else {
+                    /**
+                     * regular mode
+                     */
+                    this.__propertiesObject__.__elementOverrides__.value[name] = customCommand
+                }
+            } else if (client[name] && origCommands[name]) {
+                delete client[name]
+                unit.lift(name, customCommand, proto, (...args) => origCommands[name].apply(this, args))
+            } else {
```

Comment:
if we return above then we don't need an else and save in indentation

Reply:
in such case we need two return statements in `if` and `else if` blocks.
Is it better this instead of `else` block? 
Don't mind change it, just want to double check. 
@christian-bromann